assets/scss/03-organisms/_utility-nav.scss

```diff
@@ -107,7 +107,7 @@ $utility-nav-height: 43px;
       text-align: left;
     }
 
-    @media ($bp-header-toggle-max) {
+    @media ($bp-header-toggle-max) {// I MIGHT REMOVED IT TO REMOVE THE CHEVRON.

```

Comment:
Did you end up needing to remove this to remove the chevron?

Reply:
It turned out that I was able to override the necessary styles to remove the chevron. So, keeping this not to interfere the react's styles since this files share between patternlab and react.  I removed the comment.